lms/static/scripts/postmessage_json_rpc/server/index.js

```diff
@@ -1,15 +1,32 @@
 import Server from './server';
-import { requestConfig, requestGroups } from './methods';
+import { requestConfig } from './methods';
```

Comment:
I would recommend to just move the remaining contents of `methods` into this file, since it is 4 lines of trivial code and I don't see any value in having a separation between `index.js` and `methods.js`.

As an aside, Jon B had to do a similar refactoring on the backend recently.

Reply:
makes sense